holidays/countries/iraq.py

```diff
@@ -0,0 +1,301 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+# References -> https://en.wikipedia.org/wiki/Public_holidays_in_Iraq
+# https://www.timeanddate.com/holidays/iraq/
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars.gregorian import (
+    JAN,
+    FEB,
+    MAR,
+    APR,
+    MAY,
+    JUN,
+    JUL,
+    AUG,
+    SEP,
+    OCT,
+    NOV,
+    DEC,
+    FRI,
+    SAT,
+)
+from holidays.groups import (
+    InternationalHolidays,
+    IslamicHolidays,
+    PersianCalendarHolidays,
+    ChristianHolidays,
+    HebrewCalendarHolidays,
+)
+from holidays.holiday_base import HolidayBase
+
+
+class Iraq(
+    HolidayBase,
+    InternationalHolidays,
+    IslamicHolidays,
+    PersianCalendarHolidays,
+    ChristianHolidays,
+    HebrewCalendarHolidays,
+):
+    country = "IQ"
+    default_language = "ar_IQ"
+    estimated_label = tr("%s (تقديري)")
+    supported_languages = ("ar_IQ", "en_US")
+    start_year = 1933
+    weekend = {FRI, SAT}
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
```

Comment:
If method has specific parameters, here should be docstring with description (see other countries with Islamic holidays).